spec/models/category_spec.rb

```diff
@@ -3,4 +3,39 @@
 describe Category do 
   it { should have_many(:videos) }
 
+  describe "#recent_videos" do
+
+    it "returns videos in reverse chronological order by created_at" do
+      category = Category.create(name: "Comedy")
+      futur = Video.create(title: "Futurama", description: "Space travel", category: category, created_at: 1.day.ago)
+      south = Video.create(title: "South Park", description: "Fun and games", category: category)
+      expect(category.recent_videos).to eq([south, futur])
+    end
+
+    it "returns all videos in reverse chronological order if less six videos" do 
+      category = Category.create(name: "Comedy")
+      futur = Video.create(title: "Futurama", description: "Space travel", category: category, created_at: 1.day.ago)
+      south = Video.create(title: "South Park", description: "Fun and games", category: category)
+      expect(category.recent_videos.count).to eq(2)
```

Comment:
You are not testing the order. You should either test it or remove it from the spec description.